crates/glaredb/src/args/local.rs

```diff
@@ -57,6 +57,14 @@ pub struct LocalClientOpts {
     /// Max number of rows to display.
     #[arg(long)]
     pub max_rows: Option<usize>,
+
+    /// Path to CA certificate for TLS (required for TLS protocol).
+    #[arg(long)]
+    pub ca_cert_path: Option<String>,
```

Comment:
this is for the client (I am, admittedly new so "client for what?" might be a thing...  shouldn't/wouldn't things fall back to use system certificates, and using this for specific certificates? (presumably in non-testing environments you'd use letsencrypt or something similar.)

In general having the commands/operations have "tls by default, have a flag to disable it" is a good idea.

Reply:
updated the help line to mention that this is for rpcsrv proxy TLS